src/git.zsh

```diff
@@ -90,71 +92,36 @@ _fzf_complete_git() {
             return
         fi
 
-        if [[ $prefix =~ '^--(fixup|reedit-message|reuse-message|squash)=' ]]; then
-            prefix_option=${prefix/=*/=} _fzf_complete_git-commits '' $@
+        if [[ $(_fzf_complete_git_has_options 'c C fixup reedit-message reuse-message squash' $last_argument $prefix) = 0 ]]; then
```

Comment:
How about changing the parameter order of `_fzf_complete_git_has_options` to:

- $1: last_argument
- $2: prefix
- ${@:3}: options

so that its invocation looks like the following:
```zsh
_fzf_complete_git_has_options "$last_argument" "$prefix" c C fixup reedit-message reuse-message squash
```

Reply:
Is it hard to understand that the options are behind the arguments?

Reply:
No.